libraries/Wire/src/Wire.cpp

```diff
@@ -146,6 +146,11 @@ uint8_t TwoWire::requestFrom(uint8_t address, uint8_t quantity, uint32_t iaddres
 
     // perform blocking read into buffer
     uint8_t read = 0;
+    if (sendStop != false) {
```

Comment:
`sendStop` is an `uint8_t ` so use 0 instead of bool type.
```suggestion
    if (sendStop != 0) {
```

Reply:
Most of Arduino libraries uses the requestFrom routines by passing a bool argument in SendStop position.
I agree that the sendStop type is uint8_t, but in the routine `twi_readFrom` from the Arduino framework says that : `sendStop: Boolean indicating whether to send a stop at the end`.
How to be consistent with the legacy framework ?

Reply:
Even if library use true/false value this will not change behavior.
And this is consistent with legacy framework as prototype is the same